CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing Guide
+
+## How to Contribute
+1. Find a [issue tagged with "help wanted"](https://github.com/TheOdinProject/theodinproject/labels/Help%20Wanted) to work on. Please comment on the issue to let others know you are working on it.
+
+2. Fork this repo on your local machine.
+
+	* Fork and clone the this to your local machine, use [this guide](https://help.github.com/articles/fork-a-repo/) if you don't know how to do that
+	* Set the upstream remote so you can keep your copy of the app synced with the original. To do that go to your terminal and `cd` into your cloned curriculum directory. Then use one of the following commands:
+
+	If you have ssh set up with Git
+	```
+	$ git remote add upstream git@github.com:TheOdinProject/curriculum.git
+	```
+	Otherwise
+	```
+	$ git remote add upstream https://github.com/TheOdinProject/curriculum.git
+	```
+
+3. Before you start working on your issue create a branch and name it like the following examples:
+
+	If its a new feature
+	```
+	$ git checkout -b improvement/improvement-name`
```

Comment:
The ultimate nitpick 😆   but can we change this to feature instead: `feature/feature-name`